src/views/mfa-verify/SendEmailAndVerifyCodeForm.js

```diff
@@ -42,24 +42,30 @@ define([
     events: Object.assign({}, Okta.Form.prototype.events, {
       submit: function (e) {
         e.preventDefault();
-        this.clearErrors();
+        this.handleSubmit();
+      }
+    }),
+
+    handleSubmit () {
+      this.clearErrors();
 
-        if (this.options.appState.get('isMfaChallenge')) {
-          if (this.isValid()) {
-            this.model.save();
-          }
-        } else {
-          // Send email and switch to verification view
-          this.model.set('answer', '');
-          this.model.save()
-            .then(this.renderChallengView.bind(this));
+      if (this.options.appState.get('isMfaChallenge')) {
+        if (this.isValid()) {
+          this.model.save();
         }
+      } else {
+        // Send email and switch to verification view
+        this.model.set('answer', '');
+        this.model.save()
+          .then(this.renderChallengView.bind(this));
       }
-    }),
+    },
 
     initialize: function () {
       Okta.Form.prototype.initialize.apply(this, arguments);
 
+      this.handleSubmit = _.throttle(this.handleSubmit, 100, { trailing: false });
```

Comment:
Credit to @haishengwu-okta , added throttle here to prevent concurrent API calls